lib/CFRMessageProvider.jsm

```diff
@@ -305,7 +305,6 @@ const CFR_MESSAGES = [
   {
     id: "PIN_TAB",
     template: "cfr_doorhanger",
-    exclude: true,
```

Comment:
do we want to get this "unexcluded" with this patch or should I undo this change?

Reply:
undo it, we'll turn it on with a separate PR.